core/src/androidTest/java/in/testpress/core/database/CommentDaoTest.kt

```diff
@@ -0,0 +1,34 @@
+package `in`.testpress.core.database
+
+import `in`.testpress.database.entities.CommentEntity
+import `in`.testpress.database.entities.ContentObject
+import `in`.testpress.database.entities.ProfileDetails
+import `in`.testpress.util.getOrAwaitValue
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.junit.Assert
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class CommentDaoTest: DbTestMixin() {
+
+    @Test
+    fun readShouldReturnInsertedData() {
+        insertCommentIntoDb()
+
+        val fetchedComment = db.commentDao().getAll().getOrAwaitValue()
+        Assert.assertEquals(commentFixture(),fetchedComment)
+    }
+
+    private fun insertCommentIntoDb() {
+        commentFixture().forEach {
```

Comment:
Can you also insert profile details and content object

Reply:
Since the table is one to one we will use embedded which will join content entity's and profile details's  column as a column of Comment Entity. So when we insert the data to the comment Entity it will insert the data for content and profile Details.